man/fi_efa.7.md

```diff
@@ -13,7 +13,18 @@ fi_efa \- The Amazon Elastic Fabric Adapter (EFA) Provider
 
 The EFA provider supports the Elastic Fabric Adapter (EFA) device on
 Amazon EC2.  EFA provides reliable and unreliable datagram send/receive
-with direct hardware access from userspace (OS bypass).
+with direct hardware access from userspace (OS bypass). For reliable
+datagram (RDM) EP type, it supports two fabric names: `efa` and `efa-direct`.
+The `efa` fabric implements a set of
+[wire protocols](https://github.com/ofiwg/libfabric/blob/main/prov/efa/docs/efa_rdm_protocol_v4.md)
+to support more capabilities and features beyond the EFA device capabilities.
+The `efa-direct` fabric, on the contrary, offloads all libfabric to
+the device directly without wire protocols. Compared to the `efa` fabric, the `efa-direct`
+fabric supports less capabilities and has more mode requirements for applications.
+But it is a cleaner interface for applications to utilize the pure device capabilities,
```

Comment:
why cleaner interface? The Libfabric APIs to use `efa` and `efa-direct` are the same?

Reply:
Removed this phrase